dace/sdfg/state.py

```diff
@@ -56,6 +56,17 @@ class StateGraphView(object):
     def __init__(self, *args, **kwargs):
         self._clear_scopedict_cache()
 
+    ###################################################################
+    # Typing overrides
+    
+    @overload
```

Comment:
Is this to assist static type analysis (mypy, etc.)?

Reply:
Yes